src/app/systelab-components/select/gender-combobox.component.ts

```diff
@@ -16,17 +16,22 @@ class Element {
 export class GenderSelect extends AbstractComboBox<Element> implements AfterViewInit {
 
 	@Input() showAll = false;
+	private _descriptionAll;
+	private _descriptionUnknown;
```

Comment:
Do not use the _ prefix. Is only for private and when you define set and get.

Reply:
Ok. Fix it